README.md

```diff
@@ -919,6 +919,42 @@ parsers, it may be more convenient to use the `xml_mode` context manager:
     # ... We have now returned to using the default XML parser.
 ```
 
+Application Cleanup
+-------------------
+
+A user-defined function may be registered to be invoked before, during, or after the time
```

Comment:
Do we want to advertise the "during" stage?

Reply:
We do not.

Reply:
Agree.